### PR TITLE
translate docs/add-a-manifest-file.md to Indonesia

### PR DESCRIPTION
**Pull request by adiatma**
Mohon di review mas @cendekia @coroo dan @dennyfauzan 🙏

### CODE REVIEW

File: docs/docs/add-a-manifest-file.md [hunk 1]
```diff
@@ -53,4 +52,4 @@ npm install --save gatsby-plugin-manifest
 }
 ```
 
-That's all you need to get started with adding a web manifest to a Gatsby site. The example given reflects a base configuration -- check out the [plugin reference](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode) for more options.
+Itu saja yang kamu butuhkan untuk menambahkan file manifest di website Gatsby. Contoh yang diberikan menggambarkan konfigurasi basis -- lebih lanjut lihat referensi di [plugin reference](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode) untuk opsi lebih lanjut.
```
Comment (coroo) [inline docs/docs/add-a-manifest-file.md]: For “plugin reference”, please change it to “referensi _plugin_”

**Review by coroo — CHANGES_REQUESTED** [review]
Great Job @adiatma . It would be great if you can help us to change a comment above. Also if you dont mind, for any english word, you can use italic text for it 😁

File: docs/docs/add-a-manifest-file.md (hunk 1 above)
Comment (adiatma) [reply docs/docs/add-a-manifest-file.md]: Thanks for the review 🙏

**Review by coroo — APPROVED** [review]
Thank you @adiatma . 
Congrats and dont forget to claim your swag for free 🥇

**Pull request merged**